src/frontends/pytorch/src/op/meshgrid.cpp

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "pt_framework_node.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_meshgrid(NodeContext& context) {
+    auto node = context.mark_node(std::make_shared<PtFrameworkNode>(context.get_decoder(), context.inputs()));
+    node->set_argument(0, context.get_input_from_visible_context(0));
```

Comment:
`context.get_input_from_visible_context` can return not only a constant. If it is not constant we cannot move it inside of the body graph. In this case you need to call `const_input<std::string>` for string constant (which will propagate constant from outside internally) or you can check if the node returned by `context.get_input_from_visible_context` is a `Constant` and in that case use it as input for framework node. Also you add inputs here to `PtFrameworkNode` while you need to create this node with such inputs just like any other operation.

Reply:
Thanks, modified translation to use const_input<std::string> for indexing input and passed it as PtFrameworkNode attribute